docs/customizations/helm.rst

```diff
@@ -421,606 +433,15 @@ NIC Configuration Operator Helm chart customization options can be found `here <
-            vendors: [15b3]
-            rdmaHcaMax: 63
-         
-     - The following defines the RDMA resources in the cluster. It must be provided by the user when deploying the chart. Each entry in the resources element will create a resource with the provided <name> and list of devices.
-   * - rdmaSharedDevicePlugin.useCdi
-     - bool
-     - `false`
-     - Enable Container Device Interface (CDI) mode. **NOTE**: NVIDIA Network Operator does not configure container runtime to enable CDI.
-   * - rdmaSharedDevicePlugin.version
-     - string
-     - `"sha-4f3eb2224b8b5f97be3f17441ddee8d41753b7d5"`
-     - RDMA shared device plugin version.
-
-==========================================
-RDMA Device Plugin Resource Configurations
-==========================================
-
-These configurations consist of a list of RDMA resources, each with a name and a selector of RDMA capable network devices to be associated with the resource. Refer to `RDMA Shared Device Plugin Selectors <https://github.com/Mellanox/k8s-rdma-shared-dev-plugin#devices-selectors>`_ for supported selectors.
-
-.. code-block:: yaml
-
-    resources:
-    - name: rdma_shared_device_a
-      vendors: [15b3]
-      deviceIDs: [1017]
-      ifNames: [enp5s0f0]
-      rdmaHcaMax: 63
-    - name: rdma_shared_device_b
-      vendors: [15b3]
-      deviceIDs: [1017]
-      ifNames: [ib0, ib1]
-      rdmaHcaMax: 63
-
-============================
-SR-IOV Network Device Plugin
-============================
-
-.. list-table::
-   :header-rows: 1
-
-   * - Name
-     - Type
-     - Default
-     - Description
-   * - sriovDevicePlugin.deploy
-     - bool
-     - `false`
-     - Deploy SR-IOV Network device plugin.
-   * - sriovDevicePlugin.image
-     - string
-     - `"sriov-network-device-plugin"`
-     - SR-IOV Network device plugin image name.
-   * - sriovDevicePlugin.repository
-     - string
-     - `"ghcr.io/k8snetworkplumbingwg"`
-     - SR-IOV Network device plugin image repository.
-   * - sriovDevicePlugin.resources[0].name
-     - string
-     - `"hostdev"`
+     - `"v0.1.4"`
      -
-   * - sriovDevicePlugin.resources[0].vendors[0]
-     - string
-     - `"15b3"`
-     -
-   * - sriovDevicePlugin.useCdi
-     - bool
-     - `false`
-     - Enable Container Device Interface (CDI) mode. **NOTE**: NVIDIA Network Operator does not configure container runtime to enable CD.
-   * - sriovDevicePlugin.version
-     - string
-     - `"v3.7.0"`
-     - SR-IOV Network device plugin version
-
-===================================================
-SR-IOV Network Device Plugin Resource Configuration
-===================================================
-
-Consists of a list of RDMA resources, each with a name and a selector of RDMA capable network devices to be associated with the resource. Refer to `SR-IOV Network Device Plugin Selectors <https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#device-selectors>`_ for supported selectors.
```

Comment:
Make sure we have similar link/description in the SR-IOV Network Device Plugin examples

Reply:
maybe a section in advanced configuration pointing to the docs of sriov device plugin ?

Reply:
+1 for godoc